src/utils/defaultKeyExtractor.ts

```diff
@@ -0,0 +1,5 @@
+type Item = any;
+
+export function defaultKeyExtractor(item: Item, index: number) {
```

Comment:
```suggestion
export function defaultKeyExtractor(item: unknown, index: number) {
```

이렇게 어떠심

unknown은 아직 타입은 알 수 없지만, item을 사용할 때는 타입을 지정하게 강제할 수 있음
unknown을 쓰면 지금처럼 item을 쓰지 않는 상황에선 any처럼 사용되지만, 만약에 item을 사용하는 코드가 생기면 타이핑을 해줘야 하도록 강제 할 수 있음